### ISSUE TITLE
DE Lagging

### ISSUE THREAD

**Issue by abadgerw** [daisybio/PRONE.Shiny#12]
Thank you for all your hard work! I tried running DE analysis and while ROTS worked, LIMMA still had the previous issue of lagging and never completing.

**Comment by abadgerw** [comment]
Just wanted to follow up on this @lisiarend?

**Comment by lisiarend** [comment]
Hey @abadgerw,

I apologize for the delay. I was on vacation, and we're currently managing some deadlines. It's on my todo list, and I'll aim to review it this week.

**Comment by lisiarend** [comment]
Hey @abadgerw,
the problem is that your conditions are numbers. And they have to be syntactically valid names in R. A very quick workaround would be to replace the values 1 and 2 in your Status column by strings such as cond1 and cond2 for example.

It is difficult to make this automatically because then by default your conditions will be named X1, X2, etc. I would rather like to print the error message to the user and the user changes the names than changing the names by default to X1, X2. These names will then also be used in the DE plots which then does not fit with condition naming of the previous plots.

**Comment by abadgerw** [comment]
Thank you for looking into this! Just to confirm, all I have to do is change the numeric designations into characters for my DE groups?

**Comment by lisiarend** [comment]
Yes exactly. Change the numeric designations of your sample groups in the metadata file.

If the problem persists, feel free to contact me again. This time I will answer more quickly.

Best,
Lis
